Formula/ooniprobe.rb

```diff
@@ -135,12 +169,15 @@ class Ooniprobe < Formula
   def install
     ENV["PYTHONPATH"] = Formula["libdnet"].opt_lib/"python2.7/site-packages"
     ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+
+    # We need to set the PATH prefix otherwise ooniprobe will not manage to
+    # find homebrew installed tor
+    ENV.prepend_create_path "PATH", "#{HOMEBREW_PREFIX}/bin"
```

Comment:
It seems like it should be configured to do this in the plist and use `"#{HOMEBREW_PREFIX}/opt/tor/bin"`.